posts/Yoann/2020-12-11-hibernate-search-6-0-0-Final.adoc

```diff
@@ -0,0 +1,295 @@
+
+More consistent::
+* The API types consistently use the `Search` prefix: no more mixing `FullText` with `Search` or simply no prefix.
+* The `SearchQuery` type (previously `FullTextQuery`)
+now link:{hsearch-doc-url-prefix}#search-dsl-query-fetching-results[defines its own methods]
+instead of extending JPA's `TypeQuery`,
+highlighting the fact that it targets an index and not a database, and making it a friendlier API.
+It is still possible to
+link:{hsearch-doc-url-prefix}/#search-dsl-query-object[create an adapter that implements JPA's `TypeQuery`].
+
+Backend-agnostic by default, with backend-specific extensions::
+* As explained in <<api-refresh>>, the Search DSL no longer exposes Lucene-specific types by default.
+Native Lucene queries can still be used (see below).
+* Injection of native predicates
+(link:{hsearch-doc-url-prefix}#search-dsl-predicate-extensions-lucene-from-lucene-query[`org.apache.lucene.search.Query` for Lucene],
+link:{hsearch-doc-url-prefix}#search-dsl-predicate-extensions-elasticsearch-from-json[JSON for Elasticsearch])
+within DSL-created predicates.
+This is not new for the Lucene integration, but it is for the Elasticsearch integration.
+See link:{hsearch-doc-url-prefix}#search-dsl-predicate-extensions[predicate extensions].
+* Injection of native link:{hsearch-doc-url-prefix}#search-dsl-sort-extensions[sorts]
+and link:{hsearch-doc-url-prefix}#search-dsl-aggregation-extensions[aggregations]
+within DSL-created Hibernate Search queries.
+* Ability to link:{hsearch-doc-url-prefix}#search-dsl-query-elasticsearch-json[directly manipulate the JSON of Elasticsearch requests/responses].
+
+More features::
+* A new link:{hsearch-doc-url-prefix}#search-dsl-predicate-id[`id` predicate].
+* A new link:{hsearch-doc-url-prefix}#search-dsl-predicate-exists[`exists` predicate].
+* Ability to link:{hsearch-doc-url-prefix}#search-dsl-predicate-common-overriding-analysis[override analyzers on a per-predicate basis].
+* Better for support for multi-valued fields for link:{hsearch-doc-url-prefix}#search-dsl-sort-common-multi-value-mode[sorts],
+link:{hsearch-doc-url-prefix}#search-dsl-projection-field-multivalued[projections]
+and link:{hsearch-doc-url-prefix}#search-dsl-aggregation[aggregations (faceting)].
+
+[[mapping]]
+=== Easier mapping
+
+Specialized `@*Field` annotations::
+Hibernate Search 6 offers separate `@*Field` annotations for each major field type:
+link:{hsearch-doc-url-prefix}#mapper-orm-directfieldmapping-annotations[`@GenericField`] works in most cases,
+but you can access more options (analysis, ...)
+with link:{hsearch-doc-url-prefix}#mapper-orm-directfieldmapping-annotations-fulltextfield[`@FullTextField`],
+link:{hsearch-doc-url-prefix}#mapper-orm-directfieldmapping-annotations-keywordfield[`@KeywordField`],
+...
++
+This way, each annotation offers the options that make sense for the selected field types,
+and only those options. No longer will you end up disabling analysis on a numeric field!
```

Comment:
I would add a conjunction phrase before the `No longer ...`.
E.g.: `As a consequence no longer will ...`.